src/lib/validator/schemas.ts

```diff
@@ -0,0 +1,9 @@
+export default {
+  signUp: {
+    firstName: 'required|string|min:2|max:20',
+    lastName: 'required|string|min:2|max:20',
+    email: 'required|email|max:20',
```

Comment:
email length should be 320

Reply:
Also, passwords are meant to allow symbols so the `alpha_dash` is not necessary @chukwuemekachm